x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_expression.tsx

```diff
@@ -17,9 +17,10 @@ import {
   BarSeries,
   Position,
 } from '@elastic/charts';
+import { npStart } from 'ui/new_platform';
```

Comment:
We've been following a rule of not allowing ourselves to import from new platform except at the top level of each plugin so that we stay ready for new platform migration. Instead, we should be importing dependencies like this in the `xy_visualization_plugin/plugin.ts` file and passing them down.

While that is what we should be doing to be most correct, the XY visualization isn't actually set up to take imports like this today because it doesn't have a `start` phase. It only has a `setup` phase, and the XY visualization plugin itself is static.

Here's where I think this lands:

* Unless we break our import rule, there isn't a way for Lens visualizations to use dependencies from new platform
* The most correct solution involves restructuring some of the Lens initialization code from the top, but I don't actually think this is a time-consuming change.
* I will start on the restructuring I mention above

While I don't want to break our rule about imports, I also don't want to block your work. So I think this goes back to you: I think I can implement the "real" fix for next week. Can you wait on integrating EUI styles until next week?

Reply:
Hey @wylieconlon. Just a reminder that @cchaos is out on vacation starting mid next week. Is this something you can possibly do on your own in a separate PR if we merged this in? Just want to be aware of timing. That leaves it a little less on us to keep track of the follow up work.

Reply:
As @snide said, since I'll be out starting next week, I'd rather get this in since it **is** working. And then you all can follow up with the _proper_ way.

Reply:
Oh I see now, the tests are failing on `EuiUtils`. That's really odd since this is the same exact setup as used in the [Watcher visualization](https://github.com/elastic/kibana/pull/45446/files#diff-75c6448160b4ff9bba87feb6e58ca2a8) and there's no build or console errors.

Reply:
I'm with Wylie on this one. Wylie or I can always take over this and complete it while you're out. I can also take a look at it on Monday and also look into the failing test while I'm at it.